docs/setup-and-installing.md

```diff
@@ -1,153 +1,81 @@
 **Warning:** The alpha release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../README.md#recommended-use).
 
 -----
+# Set up the SpatialOS GDK for Unity
 
-# Set up and get started with the SpatialOS GDK for Unity
+## Quick version
 
-## Short version
-
-1. Set up your machine by installing:
-
-* (For Windows) [Unity 2018.2.0f2](https://unity3d.com/get-unity/download/archive) with Linux and Mac build support, [Visual Studio 2017](https://www.visualstudio.com/downloads/), [.NET Core SDK (x64)](https://www.microsoft.com/net/download/)  (verified with `2.1.3xx` and `2.1.4xx`), [SpatialOS](https://console.improbable.io/installer/download/stable/latest/win), [ReSharper](https://www.jetbrains.com/resharper/) (optional), and [ReSharper Command Line Tools](https://www.jetbrains.com/resharper/download/index.html#section=resharper-clt) (optional).
-* (For Mac) [Unity 2018.2.0f2](https://unity3d.com/get-unity/download/archive) with Linux and Windows build support, [Rider](https://www.jetbrains.com/rider/) (optional; alternatively [Visual Studio 2017](https://www.visualstudio.com/downloads/)), [.NET Core SDK (x64)](https://www.microsoft.com/net/download/) (verified with `2.1.3xx` and `2.1.4xx`), [SpatialOS](https://console.improbable.io/installer/download/stable/latest/mac).
-
-1. Clone the repository: `git clone git@github.com:spatialos/UnityGDK.git`  or `git clone  https://github.com/spatialos/UnityGDK.git`
-
-1. Open the `workers/unity` project in your Unity Editor.
-
-2. Choose **SpatialOS** > **Local launch** or press **Ctrl-L**
-
-3. In the Editor, open **Assets** > **Playground** > **Scenes** > **SampleScene**.
+1. Sign up for a  [SpatialOS account](https://improbable.io/get-spatialos).
+2. Setup your machine by installing the prerequisites. See the install guide for [Windows](#windows) or [MacOS](#mac).
+3. Clone the UnityGDK repository: `git clone git@github.com:spatialos/UnityGDK.git` or `git clone https://github.com/spatialos/UnityGDK.git`
+4. Follow the [Build and Deploy](./content/build-and-deploy.md) guide to run the example `Playground` project.
```

Comment:
Link to the Playground title of the deploy.md file.

Reply:
Yup - am waiting for Jess's PR to get finalised and merged 👍